hack/local-development/start-apiserver

```diff
@@ -57,7 +57,6 @@ apiserver_flags="
   --tls-cert-file $TLS_CERT_FILE \
   --tls-private-key-file $TLS_KEY_FILE \
   --feature-gates SeedChange=true \
-  --feature-gates AdminKubeconfigRequest=true \
```

Comment:
Deletion of this line is not required because with the beta version user has the access to enable or disable the feature gate.

Reply:
If user can change this to false, they can as well add the line.